rs/nns/integration_tests/src/wait_for_quiet.rs

```diff
@@ -48,6 +54,7 @@ fn test_deadline_is_extended_with_wait_for_quiet() {
                 controller: Some(neuron_4_owner_principal_id),
                 cached_neuron_stake_e8s: 200_000_000,
                 dissolve_state: Some(DissolveState::DissolveDelaySeconds(ONE_DAY_SECONDS * 365)),
+                voting_power_refreshed_timestamp_seconds: Some(now_timestamp_seconds),
                 ..Default::default()
```

Comment:
This later (i.e. now) masked the problem. This shows why we should try (harder) to avoid this, even though it is often very convenient.